lib/driver.js

```diff
@@ -48,7 +49,7 @@ function Setup() {
         //are we running the tests on the local machine?
         if (localServer === true) {
           log('test locally');
-          if (tgtBrowser !== 'chrome' && tgtBrowser !== 'phantomjs') {
+          if (tgtBrowser !== 'chrome' && tgtBrowser !== 'phantomjs' && tgtBrowser !== 'firefox') {
```

Comment:
my initial thought was this could be an array, and the if check could be `targets.indexOf(tgtBrowser) === -1`, but that's a small nitpick (and requires ES5).  I'm not even sure of the nemo2 node version support.  I guess that's not gonna change until nemo v3?